taproot-updates.md

```diff
@@ -284,13 +283,13 @@ structure (or something similar) to our commitment transaction:
     # funds go back to us via a second-stage PTLC-timeout transaction (which contains an absolute delay)
     # NB: we need the remote signature, which prevents us from unilaterally changing the PTLC-timeout transaction
     <remote_ptlcpubkey> OP_CHECKSIGVERIFY <local_ptlcpubkey> OP_CHECKSIG
-    1 OP_CHECKSEQUENCEVERIFY
+    OP_CHECKSEQUENCEVERIFY
   ",
   "tapleaf_2": "
     # funds go to the remote node via a second-stage Claim-PTLC-success transaction by completing an adaptor sig, revealing the payment secret
     # NB: we don't use musig2 here because it would force local and remote signatures to use the same sighash flags
     <local_ptlcpubkey> OP_CHECKSIGVERIFY <remote_ptlcpubkey> OP_CHECKSIG
```

Comment:
why are there two tapleaves when the only difference between the two is the order? one can be removed to save 32 vbytes

Reply:
this is only for readability, right?